docs/sources/flow/reference/components/loki.write.md

```diff
@@ -128,6 +130,18 @@ enabled, the retry mechanism will be governed by the backoff configuration speci
 
 {{< docs/shared lookup="flow/reference/components/tls-config-block.md" source="agent" version="<AGENT VERSION>" >}}
 
+### queue_config block (experimental)
+
+The optional `queue_config` block configures, when WAL is enabled (see [Write-Ahead block](#wal-block-experimental)), how the
+underlying client queues batches of logs to be sent to Loki.
+
+The following arguments are supported:
+
+| Name            | Type       | Description                                                                                                                                                                      | Default | Required |
+| --------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | -------- |
+| `capacity`      | `string`   | Controls the size of the underlying send queue buffer. This setting should be considered as a worst case scenario of memory consumption, in which all enqueued batches are full. | `10MiB`  | no       |
+| `drain_timeout` | `duration` | Configures the maximum time the client can take to drain the send queue, upon shutdown. During that time it will enqueue pending batches, and drain the send queue sending each. | `"1m"`  | no       |
```

Comment:
```suggestion
| `drain_timeout` | `duration` | Configures the maximum time the client can take to drain the send queue upon shutdown. During that time, it will enqueue pending batches and drain the send queue sending each. | `"1m"`  | no       |
```